AQA_Lab_Task/Task6/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using SimpleLogger;
+using SimpleLogger.Logging.Handlers;
+using Task6.Models;
+using Task6.Services;
+
+const string datafileName = "phonedata.json";
+
+const string outputFileName = "checkout.json";
+
+Logger.LoggerHandlerManager.AddHandler(new ConsoleLoggerHandler());
+Logger.DefaultLevel = Configurator.LogLevel switch
+{
+    "Fine" => Logger.Level.Fine,
+    "Debug" => Logger.Level.Debug,
+    "Info" => Logger.Level.Info,
+    "Error" => Logger.Level.Error,
+    _ => Logger.Level.None
+};
```

Comment:
Move it from Program class. You can make a static method for this initialization.